posthog/test/base.py

```diff
@@ -157,7 +157,7 @@ def clean_varying_query_parts(query, replace_all_numbers):
     #### Cohort replacements
     # replace cohort id lists in queries too
     query = re.sub(
-        r"in\(([^,]+\.?cohort_id), \[(\d+(, ?\d+)*)]\)",
+        r"in\(([^,]*cohort_id),\s*\[(\d+(?:,\s*\d+)*)]\)",
```

Comment:
be a little more flexible here because of 
![Screenshot 2024-12-16 at 17 35 22](https://github.com/user-attachments/assets/486cb6e6-9b11-41ff-af0c-c3769645a0dc)